pick.c

```diff
@@ -649,6 +649,7 @@ tty_init(void)
 		err(1, "fopen");
 
 	setupterm((char *)0, fileno(tty_out), (int *)0);
+	tty_putp(keypad_xmit, 0);
```

Comment:
Nice find! I would prefer grouping the `tty_putp` calls and make sure to
exit from the same mode.

```diff
diff --git a/pick.c b/pick.c
index 2c23aeb..df185b9 100644
--- a/pick.c
+++ b/pick.c
@@ -655,6 +655,8 @@ tty_init(void)
 	if (use_alternate_screen)
 		tty_putp(enter_ca_mode, 0);
 
+	tty_putp(keypad_xmit, 1);
+
 	signal(SIGINT, handle_sigint);
 }
 
@@ -677,6 +679,7 @@ tty_restore(void)
 	tcsetattr(fileno(tty_in), TCSANOW, &original_attributes);
 	fclose(tty_in);
 
+	tty_putp(keypad_local, 1);
 	tty_putp(carriage_return, 1);	/* move cursor to first column */
 	tty_putp(clr_eos, 1);
 
```